README.md

```diff
@@ -81,38 +146,59 @@ will be processed to:
 }
 ```
 
-## Pseudo class and elements
+Psuedo-classes are also dealt with correctly, because it's easy to statically determine.
 
 ```css
 .foo {
-	--foo-color: #ff0000;
+	--foo-color: red;
 	color: var(--foo-color);
 }
 
 .foo:hover {
-	--foo-color: #00ff00;
+	--foo-color: green;
 }
 ```
 
-will be processed to:
+The above will be transformed to this:
 
 ```css
 .foo {
-	color: #ff0000;
+	color: red;
 }
 
 .foo:hover {
-	color: #00ff00;
+	color: green;
 }
 ```
 
+It also works very well with `calc()`.
+
+```css
+:root {
+	--page-margin: 1em;
+}
+
+/* Easy centering */
+.wrapper {
+	width: calc(100% - 2 * var(--page-margin));
+	margin-left: var(--page-margin);
+	margin-right: var(--page-margin);
+}
+```
 
+The above would transform to this:
 
-## Nested rules
+```css
+.wrapper {
+	width: calc(100% - 2 * 1em);
+	margin-left: 1em;
+	margin-right: 1em;
+}
+```
 
-When using this feature, `postcss-css-variables` will output invalid CSS by itself(but you did input invalid CSS anyway). This feature is best paired with [`postcss-nested`](https://github.com/postcss/postcss-nested) in order to properly expand the rules.
+### Nested Rules
 
-Run `postcss-nested` before `postcss-css-variables` so that `postcss-nested` can properly expand the `&` references before we start resolving variable values.
+This pairs very well with [`postcss-nested`](https://github.com/postcss/postcss-nested) or [`postcss-nesting`](https://github.com/jonathantneal/postcss-nesting), adding support for nested rules. For either one, you must put it before `postcss-css-variables` so that the `&` references can be expanded first (`postcss-css-variables` doesn't understand them). For example, with `postcss-nested`, your JS can look like this:
```

Comment:
Done with a few edits (I said "you must use that plugin before `postcss-css-variables` in your setup" instead of "you must put it before `postcss-css-variables` in the plugin stack").